packages/fluent-ui/src/CheckboxWidget/CheckboxWidget.tsx

```diff
@@ -90,7 +91,7 @@ export default function CheckboxWidget<
       <Checkbox
         id={id}
         name={id}
-        label={displayLabel ? label || schema.title : undefined}
+        label={labelValue(label || undefined, hideLabel)}
```

Comment:
Of course, there is an exception, fluent does treat empty string differently than undefined here, made it more explicit